Submitted By:            Randy McMurchy <randy_at_linuxfromscratch_dot_org>
Date:                    2004-09-01
Initial Package Version: 0.7.38
Upstream Status:         Not submitted 
Origin:                  Self-created
Description:             Fixes build problem with FreeType version >2.15

$LastChangedBy$
$Date$

diff -Naur avifile-0.7-0.7.38-orig/lib/video/sub_ft.cpp avifile-0.7-0.7.38/lib/video/sub_ft.cpp
--- avifile-0.7-0.7.38-orig/lib/video/sub_ft.cpp	2002-11-01 11:08:06.000000000 +0000
+++ avifile-0.7-0.7.38/lib/video/sub_ft.cpp	2004-09-01 18:35:14.428759536 +0000
@@ -6,6 +6,9 @@
 
 #ifdef HAVE_LIBFREETYPE
 
+#include <ft2build.h>
+#include FT_FREETYPE_H
+
 #include <freetype/freetype.h>
 
 #if (FREETYPE_MAJOR > 2) || (FREETYPE_MAJOR == 2 && FREETYPE_MINOR >= 1)
